src/aead/aead.rs

```diff
@@ -296,23 +297,51 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
+    pub fn test_aes_gcm_128_slow() {
```

Comment:
Instead of `_slow`, let's use a suffix like `_in_prefix_len_extra` or another self-documenting name.